app/src/main/java/com/example/admin/personallibrarycatalogue/AddBookActivityFragment.java

```diff
@@ -0,0 +1,192 @@
+import android.net.Uri;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+import com.example.admin.personallibrarycatalogue.data.Book;
+import com.example.admin.personallibrarycatalogue.data.LibraryDatabaseHelper;
+
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+
+
+/**
+ * The fragment is using for adding and editinig book
+ */
+public class AddBookActivityFragment extends Fragment {
+
+    public final int SELECT_PHOTO = 1;
+    public final int WIDTH  = 72;
+    public final int HEIGHT = 60;
+
+
+    @Nullable
+    private Integer id_;
+    private ImageView imageView_;
+
+    public AddBookActivityFragment newInstance(String title, String author) {
+        AddBookActivityFragment fragment = new AddBookActivityFragment();
+
+        // arguments
+        Bundle arguments = new Bundle();
+
+        // Using values from string.xml resources
+        Resources resources = getResources();
+        arguments.putString(String.format(resources.getString(R.string.title_settings)), title);
+        arguments.putString(String.format(resources.getString(R.string.author_settings)), author);
+        fragment.setArguments(arguments);
+        return fragment;
+    }
+
+    public AddBookActivityFragment() {
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+
+        final View rootView = inflater.inflate(R.layout.fragment_add_book, container, false);
+        final LibraryDatabaseHelper helper = new LibraryDatabaseHelper(getActivity());
+
+        Bundle extras = getActivity().getIntent().getExtras();
+
+        if (extras != null) {
+            id_ = getArguments().getInt("id");
+            Book book = helper.getBookById(id_);
+            fillAllFields(rootView, book);
+        }
+
+        imageView_ = (ImageView) rootView.findViewById(R.id.cover_image_view);
+
+        // Actions for change cover button
+        Button setImageButton = (Button) rootView.findViewById(R.id.set_image_button);
+
+        setImageButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent photoPickerIntent = new Intent(Intent.ACTION_PICK);
+
+                ComponentName activity = photoPickerIntent.resolveActivity(getActivity().getPackageManager());
+                photoPickerIntent.setType("image/*");
+
+                if (activity != null) {
+                    startActivityForResult(photoPickerIntent, SELECT_PHOTO);
+                } else {
+                    Toast.makeText(getActivity().getBaseContext(), "There are no activities for such intent",
```

Comment:
Change text to "There are no applications to pick a photo". Also move the string to resources.